.github/workflows/ci-pipeline.yaml

```diff
@@ -107,7 +107,7 @@ jobs:
           helm install happy-panda bitnami/wordpress --namespace default
       - name: Deploy AKS Periscope
         run: |
-          (cd ./deployment && kustomize edit set image aksrepos.azurecr.io/staging/aks-periscope=localhost:5000/periscope:foo)
+          (cd ./deployment && kustomize edit set image mcr.microsoft.com/aks/periscope=localhost:5000/periscope:v1)
```

Comment:
question:  since kubelet will cache the image by tag, will it be better to tag the image with a unique name? Maybe we can use the `github.run_id` as tag here? https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
Sounds good let me try it out and will push the change ☕️❤️

Reply:
Please Note :) pushed these changes now. rest will take care in dedicated PR for test general health improvement \ refactors. ❤️☕️ Thanks you so much.